app/assets/v2/js/pages/new_bounty.js

```diff
@@ -285,7 +290,7 @@ $(document).ready(function() {
             auto_approve_workers: !!data.auto_approve_workers
           },
           hiring: {
-            hiringRightNow: data.hiringRightNow,
+            hiringRightNow: !!data.hiringRightNow,
```

Comment:
why do we do this btw?

Reply:
Not sure if was from some requirements discussed in the original hackathon PR, most of the code comes from there.

Reply:
Ah let's keep it as it was cause `!!data.hiringRightNow` is the same as `data.hiringRightNow`  right ?

Reply:
Yes. It just converts everything to True or False. Like if the value returns `null` or `undefined`, `!!` will change that to `false`. Hence we do that.

Reply:
Oh niceee